backend/packages/Upgrade/src/api/controllers/ExperimentClientController.ts

```diff
@@ -1014,7 +1014,7 @@ export class ExperimentClientController {
   @Delete('clearDB')
   public async clearDB(@Req() request: AppRequest): Promise<string> {
     // if DEMO mode is enabled, then clear the database:
-    if (env.app.demo) {
+    if (!env.app.demo) {
```

Comment:
why this change?

Reply:
please remove. i have to assume someone was doing this in debugging and accidentally committed it, if this change snuck into prod, anyone with a token would be able to delete our database by doing `DELETE /clearDB`. It makes me nervous even having this endpoint exist in the first place for this exact reason.

Reply:
code updated